models/pcc.py

```diff
@@ -2,37 +2,38 @@
 from util.database import db
```

Comment:
Consider using a more telling name like "per_capita_consumption" for the module instead of using another abbreviation.